pkg/anago/stage.go

```diff
@@ -225,6 +232,125 @@ func (d *defaultStageImpl) GenerateChangelog(options *changelog.Options) error {
 	return changelog.New(options).Run()
 }
 
+func (d *defaultStageImpl) GenerateVersionArtifactsBOM(options *spdx.DocGenerateOptions) error {
+	_, err := spdx.NewDocBuilder().Generate(options)
+	return errors.Wrap(err, "generating bill of materials")
+}
+
+// ListArtifacts is a function lists the release artifacts, will be
+// replaced once the supported platforms code is ready
+func (d *defaultStageImpl) ListArtifacts(version string) (list map[string][]string, err error) {
```

Comment:
Note: As discussed with @puerco, this is a temporarily function which will be moved to a different place later on.